app/index.js

```diff
@@ -7,7 +7,8 @@ angular.module(appName, [
     require('./sponsorHome'),
     require('./treatmentcenterDetail'),
     require('./treatmentcenterMap'),
-    require('./advertisement')
+    require('./advertisement'),
+	require('./addListing')
```

Comment:
please format your all you code. js and html files. I don't want to say this again...